cypress/e2e/1-workflows.cy.ts

```diff
@@ -73,4 +73,28 @@ describe('Workflows', () => {
 
 		WorkflowsPage.getters.newWorkflowButtonCard().should('be.visible');
 	});
+
+	it('should respect tag querystring filter when listing workflows', () => {
```

Comment:
Any reason we cannot test this with unit tests in `WorkflowsView.vue`? I think we should be moving away from e2e tests if test case is not for an end-to-end functionality

Reply:
Not sure whether I can test the hard reload without the e2e. Is it possible? @MiloradFilipovic

Reply:
Not really but you can mock different routes and see if filters are applied correctly.

Reply:
But, the issue when having a filter in the query string and hard-refreshing. Moving between routes worked fine!